src/components/custom/dashboard/trainer/pages/clients/components/ClientCard.jsx

```
@@ -0,0 +1,143 @@
+    return date.toLocaleDateString("en-US", {
+      month: "short",
+      day: "numeric",
+      year: "numeric",
+    });
+  };
+
+  return (
+    <div
+      className="group flex flex-col h-full overflow-hidden rounded-xl border border-zinc-800 transition-all duration-300 bg-black hover:shadow-lg hover:translate-y-[-2px] cursor-pointer"
+      onClick={() => handleViewClient(clientRequest)}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)